charts/dev/theme-park/Chart.yaml

```
@@ -0,0 +1,28 @@
+apiVersion: v2
+appVersion: "1.9.4"
+dependencies:
+- name: common
+  repository: https://library-charts.truecharts.org
+  version: 9.3.19
+deprecated: false
+description: Custom themes for your favorite apps!
+home: https://github.com/truecharts/apps/tree/master/charts/dev/theme.park
```

```suggestion
home: https://github.com/truecharts/apps/tree/master/charts/incubator/theme.park
```
use incubator train, dev is for ongoing large projects :)